src/5.0_to_5.1/__test_resources__/smallContentServer.ts

```diff
@@ -0,0 +1,1240 @@
+import { ContentRecord } from "@activeviam/activeui-sdk-5.0";
+
+/**
+ * Content server object, useful for tests.
+ * Lighter version of `contentServer` in `./contentServer.ts`.
+ */
+export const smallContentServer: ContentRecord = {
+  children: {
+    branches: {
+      entry: {
+        isDirectory: true,
+        owners: ["ROLE_CS_ROOT"],
+        readers: ["ROLE_CS_ROOT"],
+        timestamp: 1669226473908,
+        lastEditor: "_no_user_",
+        canRead: true,
+        canWrite: true,
+      },
+      children: {
+        __default__: {
+          entry: {
+            content: '{"owners":["ROLE_ADMIN"],"readers":["__ALL_USERS__"]}',
```

Comment:
I originally just exported the whole content server from Admin UI, so that's where it comes from. Not sure why it's `"__ALL_USERS__"` and not `"ROLE_USER"`.